"transforming-data-with-promise-chaining-solution"
1:"Check for array"
state:"failed"
body:"() => { cy.visit(baseUrl + "/main.html"); //always check this "visit" in your test case that it is set to - baseUrl + "/main.html" cy.get("#output").should("contain", ""); // First promise cy.wait(1000); cy.get("#output").should("contain", "2,4"); // Second Promise cy.wait(2000); cy.get("#output").should("contain", "4,8"); }"
displayError:"AssertionError: Timed out retrying after 4000ms: expected '<div#output>' to contain '2,4' at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/integration/tests/test.spec.js:123:23)"
